src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/auth/RegistrationService.java

```diff
@@ -34,7 +35,7 @@ public boolean register(String email, String password)
         }
 
         try {
-      User toRegister = userCreator.createUser(address, password);
+      User toRegister = userCreator.createUser(address, password, new UserSettings("settings"));
```

Comment:
```suggestion
      User toRegister = userCreator.createUser(address, password, new UserSettings());
```

Do not add an initial signature